modules/admin/admin-ui/src/main/resources/web/admin/common/js/content/site/inputtype/siteconfigurator/SiteConfigurator.ts

```diff
@@ -57,6 +69,7 @@ module api.content.site.inputtype.siteconfigurator {
             this.siteConfigProvider.onAfterPropertyChanged(() => this.ignorePropertyChange = false);
 
             this.comboBox = this.createComboBox(input, this.siteConfigProvider);
+            this.comboBox.setReadOnly(this.readOnly);
```

Comment:
Made even better solution that ensures that setReadOnly is called only 1 time after readOnly request is finished